src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/TopicPartitionWriterTest.java

```diff
@@ -83,12 +86,14 @@ public void testHandleRollFile() {
         IngestClient mockedClient = mock(IngestClient.class);
         TopicIngestionProperties props = new TopicIngestionProperties();
         props.ingestionProperties = new IngestionProperties(DATABASE, TABLE);
+        props.streaming = true;
```

Comment:
In short - to increase the test coverage. There is a condition which checks the streaming condition later in the main flow. If streaming attribute is set to true, the test case coverage further extends to cover the streaming usecases as well.

Reply:
Thanks @tanmaya-panda1   Was not sure because it just added the property to an existing test without creating a new one. Hence the confusion.  We had a similar issue in getting the telegraf PR approved for similar reasons , code and tests changes in tandem affects stability (we do not know if the existing regressions break for example). 
While it is good that increase in test coverage was attempted , may be we can run it for either cases (existing & new) either by using a parameter driven test (without code duplication)  or by adding a new test if the behavior is different.

Reply:
agreed @ag-ramachandran , I have added a new testcase with steaming enabled.